tests/js/JSStandardBearTest.py

```diff
@@ -0,0 +1,60 @@
+from bears.js.JSStandardBear import JSStandardBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file = """
+var foo = {
+  bar: 1,
+  baz: 2
+}
+var { bar, baz } = foo
+var x = 1
+function hello (arg) { return arg }
+
+if (baz === 2 && x !== 1) {
+  window.alert('hi')
+  bar = bar === 1
+    ? bar
+    : 1
+  if ((x = 33)) {
+    console.log(bar + "hello 'world'")
+  }
+} else {
+  (function myFunction (err) {
+    if (err) throw err
+    console.log('nothing')
+  })()
+  if (hello(bar)) console.log('bar')
+}
+"""
+
+bad_file = """
```

Comment:
create several `bad_file_x` variables, with different problems, and `_x` describes the problem. Like `bad_file_dict_tabs`